ga4gh/datamodel/reads.py

```diff
@@ -345,7 +372,8 @@ def convertReadAlignment(self, read):
         ret.alignment.mappingQuality = read.mapping_quality
         ret.alignment.position = protocol.Position()
         self.sanitizeGetRName(read.reference_id)
-        samFile = self.getFileHandle(self._samFilePath)
+        samFile = self._parentContainer.getFileHandle(
```

Comment:
Maybe we should pass ``samFile`` as an argument to ``convertReadAlignment`` to avoid hitting the handle cache again?